src/ExceptionRenderer/Console.php

```diff
@@ -134,12 +140,17 @@ protected function processPreviousException(): void
             return;
         }
 
-        $this->output .= \PHP_EOL . "\e[1;45mCaused by Previous Exception:\e[0m" . \PHP_EOL;
+        $this->output .= "\n" .
+            self::text('Caused by Previous Exception:', [self::BOLD, self::BG_COLOR_MAGENTA]) . "\n" .
+            self::text((string) (new static($this->exception->getPrevious(), $this->adapter, $this->exception))) .
+            self::text('--', [self::BOLD, self::COLOR_RED]);
+    }
 
-        $this->output .= (string) (new static($this->exception->getPrevious(), $this->adapter, $this->exception));
-        $this->output .= <<<'EOF'
-            \e[1;31m--
-            \e[0m
-            EOF;
+    /**
+     * Style text with ASCII colors.
+     */
```

Comment:
```suggestion
    /**
     * @param non-empty-list<self::FORMAT_*> $formats
     */
```

Reply:
please avoid too much comments as this is even completely wrong, no human nor AI would be able to understand what it really does :)